doc/classes/Image.xml

```diff
@@ -113,7 +117,7 @@
 			<argument index="3" name="format" type="int" enum="Image.Format">
 			</argument>
 			<description>
-				Create an empty image of a specific size and format.
+				Creates an empty image of a specific size and format.
```

Comment:
- specific -> given
- add:  "See [code]FORMAT_*[/code] constants."
- need to describe the "use_mipmaps" argument